source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -11,6 +11,22 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 16, 2023
+
+Concordium Client 5.1.1
+-----------------------
+
+Concordium Client has been migrated to use version 2 of the node gRPC API.
+
+- Since the node serves the V2 gRPC API on port 20000 by default, the default value of
+  the `--grpc-port` option has been updated to reflect this.
+
+- Some `raw` commands have been removed and new `raw` commands have been added.
```

Comment:
Was there a reason why the details here were removed?

```
    - `raw` family of commands:
        - Commands `raw GetTransactionStatusInBlock`, `raw StartBaker`, `raw StopBaker`,
          `raw JoinNetwork` and ` raw LeaveNetwork` have been removed.
        - Command `raw GetBlockSummary` has been removed, and replaced by the commands
          `raw GetBlockPendingUpdates`, `raw GetBlockSpecialEvents`,
          `raw GetBlockChainParameters` `raw GetBlockFinalizationSummary`. These provide
          a more granular way of accessing to the same data.
        - `raw BanNode` and `raw UnbanNode` no longer support node IDs, but now rather
          take just an IP address.
        - `raw DumpStart` takes a parameter specifying the path of the file to write
          dumped packets to, and furthermore supports a flag to specify whether raw
          packets should be written to the file.
        - Output of `raw GetBannedPeers` prints a JSON list of banned IP addresses,
          represented as strings.
        - Output of `raw GetPeerUptime` now prints an integer representing the uptime
          of the node in milliseconds.
        - Slight changes to `raw GetNodeInfo`. Notably the baker ID is now included in
          the output when the node is in the baker or finalization committee. Various
          consensus-related details about the node is also elaborated upon.
        - Slight changes to `raw GetPeerData` output. Notably the catch-up status and
          consensus-related details about the peer is elaborated upon.
        - `raw SendTransaction` no longer allows for specifying a network ID.
```

I can see it is a bit verbose and I don't necessarily mean they need to be there. But as it is written it does not provide much information. Could we link to the changelog perhaps for more details? Or I can create a release in github already with these details and we link to that?

Reply:
I removed it because it was verbose. If the details are in the changelog, I would prefer to link to that.

Reply:
I'm OK with that.